src/core/teleop_session_manager_tests/python/test_teleop_session.py

```
@@ -306,7 +306,7 @@ class MockDeviceIOSession:
     def __init__(self):
         self.update_count = 0
 
-    def update(self):
+    def update(self, step_time_ns):
```

Ruff (0.15.6)

[warning] Unused method argument: `step_time_ns`

(ARG002)